app/portainer/filters/filters.js

```diff
@@ -93,6 +93,12 @@ angular.module('portainer.app')
     return ip.slice(0, ip.indexOf('/'));
   };
 })
+.filter('homeurlformat', function () {
```

Comment:
What's the goal of this filter?

Reply:
URL format is `tcp://XXX.XXX.XXX.XXX` or `unix:///var/XXX` and may be another string for Azure.
The filter is there to remove the 'unix://' or 'tcp://' for display. We may want to move it directly as a func in homeController

Reply:
Ah then you can simply use the existing `stripprotocol` filter and remove this one.